packages/api-derive/src/bounties/bounties.spec.ts

```diff
@@ -1,16 +1,20 @@
 // Copyright 2017-2020 @polkadot/api-derive authors & contributors
```

Comment:
Ummm... not sure why this is not 2021 - https://github.com/polkadot-js/api/blob/master/packages/api-derive/src/bounties/bounties.spec.ts#L1 (Probably where the one below was copied from?)